.github/workflows/conftest.yml

```diff
@@ -0,0 +1,100 @@
+    steps:
+      # Checkout the packages repo so we can build the packages as a monorepo
+      - name: "Checkout source code at current commit"
+        uses: actions/checkout@v2
+
+      # Export the individual package's configuration as GitHub Action outputs
+      - name: "Export package config as outputs"
+        id: config
+        run: "make -C vendor/${{github.workflow}} info/github"
+
+      # Export the apk keys as files from secrets
+      - name: "Export keys"
+        if: steps.config.outputs.package_enabled == 'true'
+        run: "make -C .github/ export"
+
+      # Build the alpine packages for the matrix version of alpine
+      - name: "Build alpine packages"
+        if: steps.config.outputs.package_enabled == 'true'
+        run: "make -C vendor/${{github.workflow}} apk"
+
+      # Verify the packages were built or error
+      - name: "List packages"
+        if: steps.config.outputs.package_enabled == 'true'
+        run: 'find ${APK_PACKAGES_PATH} -type f -name \*.apk | xargs --no-run-if-empty ls -l | grep .'
+
+      # Export the artifact filename including path
+      - name: "Set output path to artifact"
+        if: steps.config.outputs.package_enabled == 'true'
+        id: artifact
+        run: 'echo "::set-output name=path::$(find artifacts/ -type f -name \*.apk)"'
+
+      # Determine which package organization we should use (e.g. dev or prod)
+      - name: "Determine package repo"
+        if: steps.config.outputs.package_enabled == 'true'
+        id: repo
+        run: |
+          if [[ ${GITHUB_REF} == 'refs/heads/master' ]]; then
+              echo "::set-output name=org::${{github.repository_owner}}"
+          else
+              echo "::set-output name=org::${{github.repository_owner}}-dev"
+          fi
+        env:
+          GITHUB_REF: ${{ github.ref }}
+
+      # Publish the artifacts
+      - name: "Push artifact to package repository"
+        if: steps.config.outputs.package_enabled == 'true'
+        id: push
+        uses: cloudsmith-io/action@v0.5.1
+        with:
+          api-key: ${{ secrets.CLOUDSMITH_API_KEY }}
+          command: 'push'
+          format: 'alpine'
+          owner: '${{steps.repo.outputs.org}}'              # Your Cloudsmith account name or org name (namespace)
+          repo: 'packages'                                  # Your Cloudsmith Repository name (slug)
+          distro: 'alpine'                                  # Your Distribution  (i.e debian, ubuntu, alpine)
+          release: 'any-version'                            # Use "any-version" if your package is compatible with more than one version of alpine linux
+          republish: 'true'                                 # Needed if version is not changing
+          file: '${{steps.artifact.outputs.path}}'          # Package filename (including path)
+          no-wait-for-sync: 'true'                          # Skip the waiting for package synchronisation (i.e. upload only)
```

Comment:
@nabadger this is wierd. not sure why, but it's missing the `debian` workflow.

Try this:

```
git pull origin master --rebase
touch .github/package-template.yml
make -C .github workflows
```

We should see this get added.